build/docker/scripts/sca.sh

```diff
@@ -56,7 +56,3 @@ echo  TODOs: `grep -r TODO * | wc -l`
 
 # LoC
 sloccount .
-
-# System Info
```

Comment:
Removing these lines looks unrelated to the task at hand, and they don't take much time, and they provide useful info.